README.md

```diff
@@ -567,6 +567,27 @@ Solutions:
 * If you are using a Swagger v2 definition, try converting it to v3 (OpenAPI)
 * Set a `maxYamlCodePoints` Java proprety with a big value, which can be passed using environment variables (via the `config.environ.envFile` config entry): `_JAVA_OPTIONS=-DmaxYamlCodePoints=99999999`
 
+### ZAP's Ajax Spider failing
+
+#### Insufficient shared memory
+
+In Linux, ZAP's Ajax Spider makes heavy use of shared memory (`/dev/shm/`). In such a case, when using the RapiDAST image, the user needs to make sure that sufficient space is available in `/dev/shm/` (by default, podman creates a 64MB directory). A size of 2G would be the minimum recommended. In podman for example, the option would be `--shm-size=2g`.
```

Comment:
`in Linux` => is it Linux only?

Reply:
I'm actually not sure how it works on MacOS. My line of thinking was:
- since the RapiDAST and ZAP images are running in a Linux container, it's the platform that matter when containers are involved
- When running on a physical host directly (either MacOS or Linux), I would expect the shared memory to be sufficient by default.

Now that I re-read that paragraph, `by default, podman creates a 64MB directory` does not look like a correct phrasing. I will re-write this.

Reply:
let's remove 'In Linux'.

Reply:
done